web/packages/teleport/src/Discover/Kubernetes/EnrollEKSCluster/EnrollEksCluster.tsx

```diff
@@ -229,11 +229,13 @@ export function EnrollEksCluster(props: AgentStepProps) {
           status: 'error',
           error: `Cluster "${selectedCluster.name}" enrollment result is unknown.`,
         });
+        emitErrorEvent('unknown error during EKS enrollment.');
```

Comment:
i think this can be more specific "unknown error during EKS enrollment: no results came back from fetching eks clusters" or something like that